src/Command/SystemSetupCommand.php

```diff
@@ -37,6 +38,13 @@ protected function configure(): void
             ->addOption('database-url', null, InputOption::VALUE_OPTIONAL, 'Database dsn')
             ->addOption('generate-jwt-keys', null, InputOption::VALUE_NONE, 'Generate jwt private and public key')
             ->addOption('jwt-passphrase', null, InputOption::VALUE_OPTIONAL, 'JWT private key passphrase', 'shopware')
+            ->addOption('cli', null, InputOption::VALUE_OPTIONAL, 'CLI based install')
+            ->addArgument('APP_ENV', InputArgument::OPTIONAL, 'Application environment')
```

Comment:
I'm not sure if these should be arguments or options. Thoughts?

Reply:
I would prefer options.